semgrep/semgrep/meta.py

```diff
@@ -339,7 +364,6 @@ def initialize_repo(self) -> None:
     def to_dict(self) -> Dict[str, Any]:
         return {
             **super().to_dict(),
-            "branch": os.getenv("GITHUB_REF"),
```

Comment:
We can delete this because the super().to_dict() already has `"branch": self.branch`, right?

Reply:
correct!